app/controllers/admin/events_controller.rb

```diff
@@ -92,7 +95,27 @@ def update
       end
     end
 
-    def create; end
+    def create
+      @url = admin_conference_program_events_path(@conference.short_title, @event)
+      @users = User.all.order(:name)
+      @languages = @program.languages_list
+      @event.submitter = current_user
+      @event.state = :confirmed
```

Comment:
I find this a bit arbitrary. I think we should allow for the evaluation process to be in place even with events created by organizers. An event might be created by an organizer long before the conference, and it makes sense that it is evaluated by the whole CfP committee before being accepted. Also I think the workflow should be that the organizer adds the real speaker of the event, and it is then the responsibility of the speaker to do the final confirmation that marks the event as confirmed. 
I realize organizer created events might be things like 'Lunch' as well, however in case it is for keynotes and such stuff it sounds safer to me if we keep the basic workflows in place. 
Others' thoughts?